uefi/src/lib.rs

```diff
@@ -115,6 +135,74 @@
 //! Contributions in the form of a PR are also highly welcome. Check our
 //! [contributing guide][contributing] for details.
 //!
+//! # Comparison to other Projects in the Ecosystem
+//!
+//! ## Rust `std` implementation
+//!
+//! There is an ongoing effort for a [`std` implementation][rustc-uefi-std] of
+//! the Rust standard library, which allows you to write UEFI programs that look
+//! very similar to normal Rust programs running on top of an OS.
+//!
+//! It is still under development. You can track the progress in the
+//! corresponding [tracking issue][uefi-std-tr-issue].
+//!
+//! Using the `std` implementation simplifies the overall process of producing
+//! the binary. For example, our [`#[entry]`][entry-macro] macro won't be
+//! required any longer. As the `std` implementation evolves over time, you'll
+//! need fewer and fewer abstractions of this crate. For everything not covered
+//! by the `std` implementation, you can obtain relevant structures to work with
+//! our crate via:
+//! - `std::os::uefi::env::boot_services()`
+//! - `std::os::uefi::env::get_system_handle()`
+//! - `std::os::uefi::env::get_system_table()`
+//!
+//! ### Example
+//!
+//! A minimal Rust app utilizing the uefi `std` implementation and leveraging
+//! functionality from `uefi` might look like as follows:
```

Comment:
Since the uefi-std functionality is still nightly-only behind a feature flag, I think it might cause confusion to show a prominent example of using it in our docs. It gives the impression that this is the right way to use the crate, but (IMO) right now it's better to use uefi-rs and stable Rust.

How about adding it as a new page under the [how-to](https://rust-osdev.github.io/uefi-rs/how_to/introduction.html) section of the book instead?

Reply:
Yes, perfect idea! Will move it there. Apart from that, is the PR fine?

Reply:
Adding a couple more comments but otherwise looks good